### ISSUE TITLE
Display a information panel in the report panel

### ISSUE THREAD

**Issue by skrishn** [Esri/solutions-components#621]
In Nearby app, there is an option to display an information panel to inform users on how to use the app. This is displayed every time a search is not performed. We got feedback to display it in the reporter app. 

Instead of displaying at all times, we can implement it on load and when clicking on the icon on the top right. 

![image](https://github.com/Esri/solutions-components/assets/3836616/ca6d3ed3-1c3e-4ef8-a39e-7210e4946600)

**Comment by chris-fox** [comment]
Isn't this already available via the Information Window option:

![image](https://github.com/Esri/solutions-components/assets/6796763/6097c85a-0384-4db0-8c21-e54a354fa479)

**Comment by skrishn** [comment]
Yes, but it opens as a modal. There was feedback to the IA team to display it in the panel.

**Comment by chris-fox** [comment]
I feel like we should push back against this request, this is the same behavior as was in the old Crowdsource Reporter, what advantage does showing it in the panel offer? The point of a terms of use is you can't get into the app until you accept.

**Comment by skrishn** [comment]
In the old reporter, we had it as a Help and users had to click OK to get into the app. But, with this request, if it is displayed at all times (after reports are submitted), the user gets to the app by clicking on the Report incident button. This is mimicking the experience in the Nearby app. Even if we show it only on app load, I don't see a solid use case or need for this. And, yes, I agree with you. I opened this issue to have it documented for future reference.
